r/R/feather.R

```diff
@@ -154,7 +154,15 @@ read_feather <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
     vars_select(names(reader), !!col_select)
   }
 
-  out <- reader$Read(columns)
+  out <- tryCatch(
```

Comment:
I see this same code repeated in 3 places. How about factoring out to a function?